connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java

```diff
@@ -105,7 +115,28 @@ private <T> void addConnectorPlugins(Collection<PluginDesc<T>> plugins, Collecti
 
     @PUT
     @Path("/{pluginName}/config/validate")
-    @Operation(summary = "Validate the provided configuration against the configuration definition for the specified pluginName")
+    @Operation(
+            summary = "Validate the provided configuration against the configuration definition for the specified pluginName",
+            parameters = {
+                    @Parameter(in = ParameterIn.PATH, schema = @Schema(implementation = StringSchema.class), name = "pluginName"),
```

Comment:
`StringSchema` doesn't get rendered as a `type: string` as I'd expect, but rather as a `StringSchema` schema object with `type: object` and a whole bunch of schema properties. I think `@Schema(type = "string")` will give you what you need.